### PR TITLE
Bugfix/fixing the velocity

### PR DESCRIPTION
**Pull request by AmishaBisht**
Targets #2256 

### Checklist:

- [ ] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/index.blade.php [hunk 1]
```diff
@@ -94,58 +94,55 @@
...
-                                        @endphp
-                                        <a class="{{ $textColor }}"
-                                            href="{{ route('project.effort-tracking', $project) }}"><i
-                                                class="mr-0.5 fa fa-external-link-square"></i></a>
-                                        <span
-                                            class="{{ $textColor }} font-weight-bold">{{ $project->velocity . ' (' . $project->current_hours_for_month . ' Hrs.)' }}</span>
+                                    @endforeach 
+                                </td>
+                                <td>
+
```
Comment (gautam071) [inline Modules/Project/Resources/views/index.blade.php:100]: INDENTATION

File: Modules/Project/Resources/views/index.blade.php [hunk 2]
```diff
@@ -66,7 +66,7 @@
                 </thead>
                 <tbody>
                     @can('projects.view')
-                        @forelse($clients as $client)
+                        @forelse($data['clients'] as $client)
```
Comment (gautam071) [inline Modules/Project/Resources/views/index.blade.php:69]: Why are you using the $data['clients'] instead of $clients

Comment (amisha296) [reply Modules/Project/Resources/views/index.blade.php:69]: I have made the changes in the service file now the data is coming under data.

File: Modules/Project/Resources/views/index.blade.php (hunk 1 above)
Comment (amisha296) [reply Modules/Project/Resources/views/index.blade.php:100]: done!

File: Modules/Project/Resources/views/index.blade.php (hunk 2 above)
Comment (gautam071) [reply Modules/Project/Resources/views/index.blade.php:69]: my question is why?

**Review by gautam071 — CHANGES_REQUESTED** [review]

File: Modules/Project/Resources/views/index.blade.php (hunk 2 above)
Comment (amisha296) [reply Modules/Project/Resources/views/index.blade.php:69]: Because data is coming from data no, I have stored the data inside the data

**Pull request closed without merging**